src/lib.rs

```diff
@@ -92,7 +91,6 @@ const LOCAL_IP: Ipv4Addr = Ipv4Addr::new(127, 0, 0, 1);
 /// assert_eq!(conf, bitcoind::Conf::default());
 /// ```
 ///
-#[non_exhaustive]
```

Comment:
I had to remove this because it prevents external crates (like `electrsd`) from being able to create a `Conf`.

Reply:
that's the entire point :)

users should use `let conf = Conf::default()` and then change the relative fields

Reply:
Ah ha! ok I removed this change.